app/presenters/field_of_operation_presenter.rb

```diff
@@ -29,20 +27,6 @@ def description
     description.html_safe if description.present?
   end
 
-  def fatality_notices
```

Comment:
I think as part of this commit we can include the presenter tests that were updated/added in commits [Migrate field_of_operation presenter test from government-frontend](https://github.com/alphagov/frontend/pull/4742/commits/5bdda531974a6981163f8a5e8060693019c7f0f1) and [Make existing presenter tests pass](https://github.com/alphagov/frontend/pull/4742/commits/13b190151b4d2f473b359cebafd76ef8b2a8bf1d)

I also think the commit [Add FieldOfOperation model](https://github.com/alphagov/frontend/pull/4742/commits/cfdb2aa91d60cce6cc63f734affd10c57db88da1) should either directly follow this commit, or is included as part of this commit, which helps follow the flow of changes.

Reply:
@deborahchua Thanks - I've made this change 👍